g3/generate/preview/cli.py

```diff
@@ -43,3 +43,32 @@ def display_selection(message, type, field="message") -> str:
         # Placeholder for AI
 
     return message
+
+
+def display_comparison(original_message: str, generated_message: str) -> None:
+    options = [
+        "Better",
+        "Worse",
+        "About the same",
+    ]
+
+    console = Console()
+
+    print("\n[-] Original commit message:\n")
+    md = Markdown(original_message, "github-dark")
+    console.print(md)
+
+    print("\n[+] Generated commit message:\n")
+    md = Markdown(generated_message, "github-dark")
+    console.print(md)
+
+    print()
+    questions = [
+        inquirer.List(
+            "selection",
+            message="Action",
```

Comment:
Should we change Action to something more suitable with the given options?
Rate output? or something equivelant?

Reply:
Added `Regenerate`, `Copy to clipboard`, and `Cancel` actions. So I think it makes sense to keep the label now.